source/agora/utils/TxBuilder.d

```diff
@@ -236,9 +236,7 @@ public struct TxBuilder
                 "Address not found in Well-Known keypairs: "
                 ~ out_ref.output.address.toString());
 
-        import Schnorr = agora.crypto.Schnorr;
-        Pair pair = Pair(ownerKP.secret, ownerKP.secret.toPoint());
-        auto sig = Schnorr.sign(pair.v, tx);
+        auto sig = ownerKP.sign(pair.v, tx);
```

Comment:
`pair.v` should be removed.

Reply:
Yes. Can we call directly from `ownerKP` ?

Reply:
Oops, fixed

Reply:
> Yes. Can we call directly from `ownerKP` ?

Yes, we can.
https://github.com/bosagora/agora/blob/39c038debbdbfd3052bb7756006a3e6d9a48dc09/source/agora/crypto/Key.d#L92-L96